datamap/bloom/src/main/java/org/apache/carbondata/datamap/bloom/BloomCoarseGrainDataMap.java

```diff
@@ -103,7 +106,19 @@ public void init(DataMapModel dataMapModel) throws IOException {
   /**
    * init field converters for index columns
    */
-  public void initIndexColumnConverters(CarbonTable carbonTable, List<CarbonColumn> indexedColumn) {
+  public void initIndexColumnConverters(CarbonTable carbonTable, String dataMapName,
+      List<CarbonColumn> indexedColumn) {
+    String cacheLevel = MapUtils.getString(
+        carbonTable.getTableInfo().getFactTable().getTableProperties(),
+        CarbonCommonConstants.CACHE_LEVEL, CarbonCommonConstants.CACHE_LEVEL_DEFAULT_VALUE);
+    this.isBlockletCacheLevel = cacheLevel.equalsIgnoreCase("blocklet");
+    if (!this.isBlockletCacheLevel) {
+      LOGGER.warn(
+          String.format("BloomFilter datamap %s runs with cache_level=block for table %s.%s,"
+              + " which may decrease its pruning performance",
```

Comment:
change to `which may decrease its pruning benefit, which lead to read more data`

Reply:
OK